src/Parser/Stores.php

```diff
@@ -15,22 +15,53 @@ public function parse($data)
         }
 
         foreach ($data as $stores) {
-            $this->results[$stores['storeIdentifier']] = $stores['id'];
+            $this->results[$stores['storeIdentifier']] = [
+                'id' => $stores['id'],
+                'defaultLanguage' => $stores['configuration']['defaultLanguage'],
```

Comment:
Apparently `defaultLanguage` isn't always set. Please check if it is set and if it is missing set an empty string as value.